chart/values.schema.json

```diff
@@ -126,10 +126,15 @@
                             "default": ""
                         },
                         "host": {
-                            "description": "The hostname for the web Ingress.",
+                            "description": "The hostname for the web Ingress. (Deprecated - renamed to `ingress.web.hosts`)",
```

Comment:
Can you add warning about deprescte options in NOTES?
https://github.com/apache/airflow/blob/c73004d0cd33d76b82b172078f572e8d4eecab39/chart/templates/NOTES.txt#L94

Reply:
Also in `UPDATING.rst`?